src/qtmozembed/declarativewebpage.cpp

```diff
@@ -65,7 +67,9 @@ DeclarativeWebPage::DeclarativeWebPage(QObject *parent)
     // subscribe to gecko messages
     std::vector<std::string> messages = { FULLSCREEN_MESSAGE,
                                           DOM_CONTENT_LOADED_MESSAGE,
-                                          LINK_ADDED_MESSAGE,
+                                          LINK_SET_ICON,
+                                          LINK_ADD_FEED,
+                                          LINK_ADD_SEARCH,
```

Comment:
I don't see where "Link:AddFeed" or "Link:AddSearch" are used. Are they planned for future use?

Reply:
Yes. This is like it was previously.

Reply:
Thanks for the explanation.